hack/lib/golang.sh

```diff
@@ -397,7 +397,7 @@ kube::golang::build_binaries_for_platform() {
         "${nonstatics[@]:+${nonstatics[@]}}"
     fi
     if [[ "${#statics[@]}" != 0 ]]; then
-      CGO_ENABLED=0 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
+      CGO_ENABLED=1 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
```

Comment:
Is the lib cross platform? If not, you'll need to separate out the bits that are platform dependent (or call a platform dependent package) into a separate file, and use build tags to include them selectively, then create a separate file the implements the same functions and use build tags to select that file for all the unsupported platforms.